data_collection/gazette/spiders/rj/rj_angra_dos_reis.py

```diff
@@ -0,0 +1,48 @@
+import re
+from datetime import date, datetime as dt
+
+import scrapy
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class RjAngraDosReisSpider(BaseGazetteSpider):
+    name = "rj_angra_dos_reis"
+    TERRITORY_ID = "3300100"
+    allowed_domains = ["angra.rj.gov.br"]
+    start_date = date(2005, 3, 11)
+
+    def start_requests(self):
+        start_year = int(self.start_date.strftime("%Y"))
+        end_year = int(self.end_date.strftime("%Y"))
+
+        for year in range(end_year, (start_year - 1), -1):
+            yield scrapy.Request(
+                f"https://angra.rj.gov.br/boletim-oficial.asp?vAno={year}"
+            )
+
+    def parse(self, response):
+        for tr in response.xpath("//tr")[1:]:
```

Comment:
Pra não ficar muito sujeito a mudanças no layout do site introduzindo uma tabela indesejada, sugiro restringir um pouco mais onde estamos esperando os `tr` aparecerem
```suggestion
        for tr in response.xpath("//article//tr")[1:]:
```

Reply:
Sugestão aplicada.